core/trino-main/src/test/java/io/trino/operator/output/BenchmarkPartitionedOutputOperator.java

```diff
@@ -175,18 +193,96 @@ public void verifyAddPage()
         public enum TestType
         {
             BIGINT(BigintType.BIGINT, 5000),
+            BIGINT_SKEWED_HASH(BigintType.BIGINT, 5000) {
+                @Override
+                public Page createPage(List<Type> types, int positionCount, float nullRate)
+                {
+                    return new Page(
+                            positionCount,
+                            createRandomBlockForType(BigintType.BIGINT, positionCount, nullRate),
+                            createRandomLongsBlock(positionCount, 2));
+                }
+            },
+
             DICTIONARY_BIGINT(BigintType.BIGINT, 3000) {
                 @Override
                 public Page createPage(List<Type> types, int positionCount, float nullRate)
                 {
-                    return createRandomDictionaryPage(types, POSITION_COUNT, nullRate);
+                    return createRandomDictionaryPage(types, positionCount, nullRate);
                 }
             },
+
             RLE_BIGINT(BigintType.BIGINT, 3000) {
                 @Override
                 public Page createPage(List<Type> types, int positionCount, float nullRate)
                 {
-                    return createRandomRlePage(types, POSITION_COUNT, nullRate);
+                    return createRandomRlePage(types, positionCount, nullRate);
+                }
+            },
+            DICTIONARY_HASH_BIGINT_20_PERCENT(BigintType.BIGINT, 3000) {
```

Comment:
what does it mean `dictionary_hash`. Neither of the columns are hash of the other one

Reply:
I'm using hash channels and partition channels interchangeably. Does `DICTIONARY_PARTITION_...` sound better?

Reply:
> Does DICTIONARY_PARTITION_... sound better?

Yup. Although I don't know what is the difference between this and `DICTIONARY_BIGINT`